api-scanner/src/test/java/net/corda/plugins/KotlinVarargMethodTest.java

```diff
@@ -24,10 +19,22 @@
     @Rule
     public TestRule rules = RuleChain.outerRule(testProjectDir).around(testProject);
 
+    private static final String[] expectedInterfaceWithVarargMethod = {
+        "public interface net.corda.example.KotlinVarargArrayMethod",
+        "  public abstract void action(String[]...)",
+        "##"
+    };
+
+    private static final String[] expectedInterfaceWithArrayVarargMethod = {
+        "public interface net.corda.example.KotlinVarargMethod",
+        "  public abstract void action(int...)",
+        "##"
+    };
+
     @Test
```

Comment:
I think you have `expectedInterfaceWithVarargMethod` and `expectedInterfaceWithArrayVarargMethod` back-to-front :wink: